docs/articles/test-workflows-high-level-architecture.md

```diff
@@ -135,6 +135,34 @@ Kubernetes doesn't have a direct mechanism to get this metadata. To obtain it, w
 
 When the image you are using is stored in the private registry, you may need to add [`pod.imagePullSecrets`](https://docs.testkube.io/articles/test-workflows-examples-basics#configuring-the-pod) to the specific Test Workflow or [**Global Template**](./test-workflow-templates#global-templates).
 
+In the case you are using private AWS Elastic Container Registry, as its [authorization tokens](https://docs.aws.amazon.com/AmazonECR/latest/userguide/registry_auth.html#registry-auth-token) are only valid for 12 hours, you can ensure persistent Testkube access by configuring a Service Account with IAM Role, follow these steps:
```

Comment:
perhaps add a "AWS Elastic Container Registry" heading to make clear that this section is specific to AWS.. !?